Diary/Diary/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+namespace Diary
+{
+     class Program
+    {
+        private static readonly DiaryService service = new();
+        static void Main(string[] args)
+        {
+            var numberTask = 0;
```

Comment:
remove this from here, there is no need to keep track of tasks number in the main program.